### PR TITLE
Added writing and removing

### PR DESCRIPTION
**Pull request by XedinUnknown**
Aims to incorporate solution for #10.

Also, fixes and enhances some things.

### CODE REVIEW

File: src/ContainerUnsetCapableTrait.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+use InvalidArgumentException;
+use OutOfRangeException;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+use stdClass;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use Psr\Container\ContainerInterface as BaseContainerInterface;
+
+trait ContainerUnsetCapableTrait
```
Comment (mecha) [inline src/ContainerUnsetCapableTrait.php]: Missing description

File: src/ContainerUnsetManyCapableTrait.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+use ArrayAccess;
+use InvalidArgumentException;
+use OutOfRangeException;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+use stdClass;
+use Traversable;
+use Dhii\Util\String\StringableInterface as Stringable;
+
+trait ContainerUnsetManyCapableTrait
```
Comment (mecha) [inline src/ContainerUnsetManyCapableTrait.php]: Missing description

File: src/NormalizeKeyCapableTrait.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Dhii\Data\Container;
+
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use InvalidArgumentException;
+use OutOfRangeException;
+
+trait NormalizeKeyCapableTrait
```
Comment (mecha) [inline src/NormalizeKeyCapableTrait.php]: Missing description

**Review by mecha — APPROVED** [review]
A few missing doc descriptions, but otherwise can't find anything wrong with it, functionally. He said, fearing he might have missed something.

*Edit:* damn it, meant to click "Request Changes"

**Comment by XedinUnknown** [comment]
@mecha, I spent a lot of time on this, and fixed a lot of things. Phan really helps determine some future problems that are not very visible to the naked eye.

**Comment by mecha** [comment]
@XedinUnknown Nice. If it's that useful, we should add it to the bootstrap.

**Comment by XedinUnknown** [comment]
@mecha, please see dhii/bootstrap#36.

**Pull request merged**